src/OnlySpans.PolyLeads.Api/Features/Seeding/SeedDocumentsCommandHandler.cs

```diff
@@ -41,6 +46,17 @@ public SeedDocumentsCommandHandler(ISender sender)
 
         foreach (var document in documents)
         {
+            var isSameDocumentInDb = await _context
+               .Documents
+               .AnyAsync(x =>
+                    x.Name == document.Name,
+                    cancellationToken);
```

Comment:
вот так удобнее читается, не надо строчки сносить

```suggestion
            var hasDocumentAlreadyStored = await _context
               .Documents
               .Where(x =>  x.Name == document.Name)
               .AnyAsync(cancellationToken);
```